.stylelintrc.json

```diff
@@ -1,11 +1,12 @@
 {
-  "extends": "stylelint-config-standard",
-  "plugins": [
-    "stylelint-scss"
-  ],
-  "ignoreFiles": ["**/*.ts", "**/*.tsx", "dist/*", "node_modules/*", "coverage/*"],
+  "extends": "stylelint-config-sass-guidelines",
```

Comment:
@krashanoff nice catch about not extending `"stylelint-config-standard"`, ill add it to the extends array!